microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -57,11 +59,19 @@ class GpuTestSuite(TestSuite):
         """,
         timeout=TIMEOUT,
         requirement=simple_requirement(
-            supported_features=[GpuEnabled(), SerialConsole],
+            supported_features=[GpuEnabled(), SerialConsole, AzureExtension],
         ),
         priority=1,
     )
-    def verify_load_gpu_driver(self, node: Node, log_path: Path, log: Logger) -> None:
+    def verify_load_gpu_driver(
+        self,
+        node: Node,
+        log_path: Path,
+        log: Logger,
+        *args: Any,
+        **kwargs: Any,
+    ) -> None:
+        _install_driver(node, log_path, log)
         _check_driver_installed(node)
```

Comment:
please invoke NvidiaSmi tool get_gpu_count in _check_driver_installed. thank you!

Reply:
add get_gpu_count and logged the output. Is that the expectation?